### PR TITLE
refactor: change the version management method

### PR DESCRIPTION
**Pull request by h-suzuki-isp**
## Description
The version of setup.py used to refer to version.py, but modified to write directly to setup.py.
And modified `ros2 caret version` command.

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Comment by h-suzuki-isp** [comment]
Because I want to use main branch,close this PR and I create new one .

File: ros2caret/verb/version.py
[the remark this answers is not in the document]
Comment (h-suzuki-isp) [reply ros2caret/verb/version.py:1]: The reason for having both `version_path` and `here_path` is to convert a relative path to an absolute path.
When using `with open`, relative paths depends on the current working directory at the time of program execution.

[the remark this answers is not in the document]
Comment (h-suzuki-isp) [reply ros2caret/verb/version.py:1]: OK, fixed in https://github.com/tier4/ros2caret/pull/169/commits/c978bb565cd58638af250e064410b53cbd68deee

File: ros2caret/verb/version.py [hunk 1]
```diff
@@ -12,13 +12,25 @@
...
 class CaretVersionVerb(VerbExtension):
 
     def main(self, *, args):
-        version = __version__
-        print(version)
+        version = self.get_version()
+        print('v' + version)
+
+    def get_version(self):
+        path = f'{os.path.dirname(os.path.realpath(__file__))}/../../setup.py'
```
Comment (ymski) [inline ros2caret/verb/version.py:28]: It would be more clear if the variable name path is changed to version_path. Can you make this modification, please?

Comment (h-suzuki-isp) [reply ros2caret/verb/version.py]: OK, fixed in https://github.com/tier4/ros2caret/pull/169/commits/8d68a1707ed47a84ce04971d45a30027a7c91d15

**Review by ymski — APPROVED** [review]
LGTM

**Pull request merged**